webapp/tests/Unit/Controller/Jury/QueueTaskControllerTest.php

```diff
@@ -144,7 +144,7 @@ public function testLazy(bool $shadowMode, int $globalLazy, int $problemLazy): v
         $config   = self::getContainer()->get(ConfigurationService::class);
         $eventLog = self::getContainer()->get(EventLogService::class);
         $dj       = self::getContainer()->get(DOMJudgeService::class);
-        $config->saveChanges(['lazy_eval_results'=>$globalLazy], $eventLog, $dj);
+        $config->saveChanges(['lazy_eval_results'=>$globalLazy], $eventLog, $dj, treatMissingBooleansAsFalse: false);
```

Comment:
I don't know why this change is needed?

Reply:
Since we don't resupply the boolean shadow mode, it would assume it is false again without this option